hal/armv7a/arch/cpu.h

```diff
@@ -49,9 +49,15 @@
 #define SYSTICK_INTERVAL 1000
 
 
+#define STACK_ALIGN 8
```

Comment:
This macro can be avoided by just assuming highest for all, i.e. 16. No harm in this.

Reply:
At first I thought about doing it like this, however, using the highest value still requires a macro or an extensive comment for readability and I think this kind of fine grained control doesn't introduce too much complication.

Reply:
I;ve decided to assume highest of all for coherency of ABI sake